codespell_lib/data/dictionary.txt

```diff
@@ -8415,7 +8433,11 @@ compliling->compiling
 compling->compiling
 complitation->compilation, complication,
 complitations->compilations, complications,
+complite->complete, compile,
+complited->completed, compiled,
```

Comment:
I wonder whether this could a misspelling for `complied`, but I doubt it can be a “voluntary” misspelling, and I cannot find a keyboard on which it could be a plausible typo. Thoughts?

Reply:
I couldn't find any examples of `complited->complied` or `complites->complies`, so I will leave these lines alone.

Reply:
I looked at a similar typo. `implited` was in [4 GitHub issue discussions](https://github.com/search?q=implited&type=issues), and `implied` was in 82900.

I think `complited->complied` is rare enough to not matter.